CHANGELOG.md

```diff
@@ -17,7 +21,7 @@ Changelog
 ### Fixed
  * thrift: Handle TStruct serialization failures gracefully (#744)
 
-## [1.15.0] - 2019-08-26
+## [1.15e0] - 2019-08-26
```

Comment:
`1.15 x 10^0`?

Reply:
whoops, fixed! hope we never have to get that high..